templates/telco/osm/index.html

```diff
@@ -85,7 +87,7 @@ <h2>Charmed OSM</h2>
         <li class="p-list__item is-ticked">Predictable release cadence and upgrade path</li>
       </ul>
       <p>
-        <a href="https://charmed-osm.com/">Read more&nbsp;&rsaquo;</a>
+        <a href="https://charmed-osm.com/">Read more about Charmed OSM&nbsp;&rsaquo;</a>
```

Comment:
*Suggestion in the copy doc* to remove this link as this site does not exist anymore.

Reply:
@Sophie-32 should this link be there?

Reply:
yes can leave off for now, PM can resubmit if they have a new link